### PR TITLE
About tweaks

### PR DESCRIPTION
**Pull request by jdepree**

### CODE REVIEW

File: about.php [hunk 1]
```diff
@@ -278,19 +278,19 @@
       						<img class="activator" width="100%" height="100%" style="border-radius: 50%;" src="images/erica.jpg">
     						</div>
     					<div class="card-content">
-      					<span class="card-title activator grey-text text-darken-4">Erica Burman, Board<i class="material-icons right">more_vert</i></span>
+      					<span class="card-title activator grey-text text-darken-4">Erica Burman, Board</span>
       						
     					</div>
     					
-    				<div class="card-reveal">
+    				<!--  <div class="card-reveal">
```
Comment (jdepree) [inline about.php]: Please delete instead of commenting out so that version control can work.

File: about.php [hunk 2]
```diff
@@ -299,19 +299,19 @@
       						<img class="activator" width="100%" height="100%" style="border-radius: 50%;" src="images/beau.jpg">
     						</div>
     					<div class="card-content">
-      					<span class="card-title activator grey-text text-darken-4">Beau Jackson, Board<i class="material-icons right">more_vert</i></span>
+      					<span class="card-title activator grey-text text-darken-4">Beau Jackson, Board</span>
       						
     					</div>
     					
-    				<div class="card-reveal">
+    				<!--  <div class="card-reveal">
```
Comment (jdepree) [inline about.php]: Same here

**Comment by jdepree** [comment]
Looks like you have 3 or 4 water charity logos on here.  Please delete the extras and review the other pictures on this ticket.  Also, please delete all commented-out code blocks.  Version Control only works properly if you actually remove the code.

**Comment by jdepree** [comment]
I didn't realize you weren't a reviewer on here - please review my comments.

**Comment by jdepree** [comment]
@malawi-mike Please finish this PR

**Comment by malawi-mike** [comment]
ok, should be good to go. thanks!

**Comment by jdepree** [comment]
Delete lines above header.inc -- this causes an error.

**Comment by jdepree** [comment]
I need you to use preview and resize these images so they match the sizes that are on the page.  For instance, house_problem is 1000x1000, but it's showing up as 220x220 on the page.  Please adjust size so it's the same as its visible size.  And be sure to maintain aspect ratio (crop as needed).

**Pull request merged**